frontend/components/ui/blocknote/blocks/quiz-multiple-choice-block.tsx

```
@@ -0,0 +1,218 @@
+          props: { options: updatedOptions as any },
+        });
+      };
+
+      const handleDelete = () => {
+        props.editor.removeBlocks([props.block]);
+      };
+
+      return (
+        <div
+          ref={blockRef}
+          onMouseDown={(e) => {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)